src/CUBLAS.jl

```diff
@@ -11,8 +11,11 @@ module CUBLAS
 
 importall Base.LinAlg.BLAS
 
-using CUDArt
-using CUDArt.rt.cudaStream_t
+using CUDAdrv: OwnedPtr, CuArray, CuVector, CuMatrix
```

Comment:
There are some methods here that used to take `CudaPtr` -- but I'm not sure exactly why. Is `Ptr` the right alternative for those methods?